src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -161,6 +162,8 @@ public OutputCommitter createCommitter(TaskAttemptContext context) throws IOExce
     private boolean allowSchemaRelaxation;
     private boolean allowSchemaRelaxationOnEmptyOutput;
 
+    private static final int BQ_IMPORT_MAX_BATCH_SIZE = 10000;
```

Comment:
Should we document this limitation in bq sink documentation?

Reply:
@CuriousVini  this limitation is addressed now via creating temp table and loading and User would not have to take any extra action.  We can mention this temp table thing in doc But I went through the Docs in this repo, we do not have a section for any such internal processing notes that does not require user action.